validateCredential.js

```diff
@@ -0,0 +1,30 @@
+const md5 = require("md5");
+
+const validateCredential = async (req, res, next) => {
```

Comment:
add apiKey as first arg, and validate that the api key in header matches with this.